rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2288,6 +2293,11 @@ def test_upgrade_db_41_to_42(user_data_dir, messages_aggregator):
             SupportedBlockchain.deserialize(x) for x in json.loads(raw_list)
         ] == [SupportedBlockchain.POLYGON_POS, SupportedBlockchain.GNOSIS]
 
+        # get current oracles and check that manualcurrent was removed and all others remain.
+        settings = db.get_settings(cursor=cursor)
+        expected_oracles = [CurrentPriceOracle.deserialize(oracle) for oracle in oracles if oracle != 'manualcurrent']  # noqa: E501
```

Comment:
generally avoid creating the variables that are used only once, like settings and expected_oracles here